src/vs/platform/theme/common/colorUtils.ts

```diff
@@ -211,6 +225,10 @@ export function getColorRegistry(): IColorRegistry {
 	return colorRegistry;
 }
 
+export const setTheme = (theme: ColorThemeData) => {
```

Comment:
that's not so nice to have this as a global function. Better add it as a function to the `IColorRegistry`. I would call it` notifyThemeUpdate(...)`